app/controllers/iiif_controller.rb

```diff
@@ -108,8 +117,8 @@ def image_info
   end
   # rubocop:enable Metrics/MethodLength
 
-  def image_params
-    params.slice(:region, :size, :rotation, :quality, :format).merge(identifier_params).merge(canonical_params)
+  def stacks_image_params
+    allowed_params.permit(:region, :size, :rotation, :quality, :format).merge(identifier_params).merge(canonical_params)
```

Comment:
I'm curious if `permit` is the right thing to use here since `allowed_params` has already permitted those, so maybe `slice` is reasonable here?